config/groovy/common.groovy

```diff
@@ -339,17 +398,26 @@ class common {
         def currentPageUrl = githubHomeApiUrl
         def slurper = new JsonSlurper()
         while (currentPageUrl) {
+            //println "currentPageUrl: $currentPageUrl"
             new URL(currentPageUrl).openConnection().with { connection ->
                 connection.content.withReader { reader ->
                     slurper.parseText(reader.text).each { item ->
                         mappedPossibleItems.put(item.name, item.description)
+                        //println "Want to get item " + item.name
                     }
                 }
                 currentPageUrl = getLink(connection, "next")
+                // TODO: This comparison is vulnerable to a page request size of "100" or anything that starts with a 1, but just using 99 above ..
+                if (currentPageUrl.contains("page=1")) {
```

Comment:
`getLink` can return null if there is no such header present. GitHub doesn't return a `Link` header if the response was not paginated, such as when there are less than 99 repositories in the organisation.
```suggestion
                currentPageUrl = getLink(connection, "next")
                // TODO: This comparison is vulnerable to a page request size of "100" or anything that starts with a 1, but just using 99 above ..
                if (currentPageUrl != null && currentPageUrl.contains("page=1")) {
```